### ISSUE TITLE
Add a role for the file upload inputs

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5136]
## Area or Component
Bulk patient upload & Test results bulk upload

## Quick Summary
The file upload input currently has a calculated role of `textbox` when running the axe dev tool.

## Accessibility Violation
[Role: The element's role is missing or incorrect](https://cdc-devtools.dequecloud.com/rule-help/en/aria-role-missing)

## Action Requested
Add a role like `input` or some other role that is more appropriate for the file upload.

## Screenshots (optional)
![Screen Shot 2023-01-25 at 22.06.24.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/91378d9d-882b-49f7-8ef5-812f9cca898f)

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by emmastephenson** [comment]
We'll need to find the correct aria role - a quick search didn't show anything specific to file selection.

**Comment by emyl3** [comment]
see @johanna-skylight's comment here: https://skylight-hq.slack.com/archives/C039689GS4S/p1675190520940539

**Comment by BobanL** [comment]
Simply adding `role="button"` causes our cypress accessibility check to fail with the error `ARIA role button is not allowed for given element`. It also seems that textbox is a suitable role for `<input type="file">` so that implicit role should be sufficient.
